src/jsonUtils/sketchImpl/findFontName.ts

```diff
@@ -94,9 +94,12 @@ export const findFont = (style: TextStyle): NSFont => {
   const defaultFontSize = 14;
 
   const fontSize = style.fontSize ? style.fontSize : defaultFontSize;
-  let fontWeight = style.fontWeight
-    ? FONT_WEIGHTS[style.fontWeight.toLowerCase()]
-    : defaultFontWeight;
+  let fontWeight =
+    style.fontWeight &&
+    String(style.fontWeight).toLowerCase() &&
```

Comment:
do we need this check? It doesn't seem necessary.
```suggestion
```

Reply:
yea...maybe the whole `fontWeight` could just be:
```suggestion
    let fontWeight = FONT_WEIGHTS[String(style.fontWeight).toLowerCase()] || defaultFontWeight;
```